vtm-android/src/org/oscim/android/AndroidAssets.java

```diff
@@ -35,6 +35,13 @@ private AndroidAssets(Context ctx) {
         mContext = ctx.getApplicationContext();
     }
 
+    @Override
+    protected String getPlatformFilePath(String file) {
+        // TODO get android asset path
+        throw new UnsupportedOperationException("Get path of Android assets currently not supported");
+        // return "file:///android_asset/" + file;
+    }
```

Comment:
Can models work with `InputStream` like all our current assets?
Then without changes the `AssetAdapter` is ready to read them.

Reply:
I don't think so. [There](https://github.com/libgdx/libgdx/wiki/Managing-your-assets)'s no mention of InputStream. 

I decided to let it handle GDX as it's the only affected platform (see new commit).

Reply:
That's unfortunate.

BTW can close this large PR and focus on next smaller ones per feature.